test/hotspot/jtreg/gc/arguments/TestParallelGCThreads.java

```diff
@@ -113,14 +113,14 @@ public static void testFlags() throws Exception {
       }
     }
 
-    // 4294967295 == (unsigned int) -1
-    // So setting ParallelGCThreads=4294967295 should give back 4294967295
+    // 2147483647 == (signed int) -1
```

Comment:
This is not technically correct: `(signed int) -1` is `-1`.
Rewrite the whole comment to: `// Test the max value for ParallelGCThreads`.